app/views/cases/filters/retention.html.slim

```diff
@@ -89,7 +89,7 @@ section#retention-cases.govuk-tabs__panel
                       td 
                         = kase.subject_name
                       td 
-                        = kase.retention_schedule.planned_destruction_date 
+                        = kase.retention_schedule.planned_destruction_date.strftime(Settings.default_date_format)
```

Comment:
As there is a default date format [defined in the locales](https://github.com/ministryofjustice/correspondence_tool_staff/blob/c6f51ee87e2413ec94964b8f26a93f7fc54704f0/config/locales/en.yml#L449) you can use `#l()` helper to localise the date:

```ruby
l(kase.retention_schedule.planned_destruction_date)
```

Reply:
Thanks I'll do that 👍